### ISSUE TITLE
Bug: Titles for new cards on board views keeps "Untitled" text

### ISSUE THREAD

**Issue by mblennegard** [mattermost/focalboard#379]
**Summary:**
The title for new cards that gets created on board views keeps the "Untitled" text as its title.
This occurs only when editing the Description text directly after the title has been updated. It does still occur if changing e.g. the Status first, then changing the Title and lastly the Description.

**Steps to reproduce the behavior:**
1. Go to a board view.
2. Click on **+ New** to add a new card.
3. Update the Title to whatever.
4. Update the Description.
5. Close the card by pressing outside the card.
6. Notice that the Title has now been reverted back to "Untitled".

**Expected behavior:**
The Title to keep the provided text.

**Screenshots:**
When taking screenshots of the issue I noticed that the title actually is saved after taking a screenshot, so it seems to save when focus is moved from the application or similar.

**Platform:**
 - Windows 10, Focalboard v. 0.6.6 (Windows Store app).

**Comment by chenilim** [comment]
Thanks @mblennegard. I am able to repro it. However, it does save correctly if, after step 4, you first press Esc to save the description. It also appears to be fixed in the current build (from source), so this should be fixed in the next release.

**Comment by mblennegard** [comment]
@chenilim Yeah, it seems to work if focus is changed from the Description field before closing the card.

Allright, good to hear!
